python/bind_data_access.h

```diff
@@ -284,8 +283,10 @@ template <class... Ts> class as_VariableViewImpl {
 
   template <class Var>
   static void set_variances(Var &view, const py::object &obj) {
+    if (obj.is_none())
```

Comment:
do we also want to support `set_variances` with a `sc.move`? I'm not sure it would be super useful, since the variances are probably already set when the data is read from Mantid.

Reply:
Indeed, I would not add this until we have a benchmark that shows that it is necessary.